admin-ui/src/pages/accounts-page.tsx

```
@@ -244,6 +243,7 @@ export function AccountsPage(): React.JSX.Element {
             <SelectContent>
               <SelectItem value="86400000">{t("accountsPage.window.last24h")}</SelectItem>
```

ESLint

[error] Replace `{t("accountsPage.window.last24h")}` with `⏎················{t("accountsPage.window.last24h")}⏎··············`

(prettier/prettier)